services/wordlist.js

```diff
@@ -2,9 +2,13 @@ const debug = require('debug')('talk:services:wordlist');
 const _ = require('lodash');
 const natural = require('natural');
 const tokenizer = new natural.WordTokenizer();
+const nameTokenizer = new natural.RegexpTokenizer({pattern: /\_/});
 const SettingsService = require('./settings');
 const Errors = require('../errors');
 
+// REGEX to prevent emoji's from entering the wordlist.
```

Comment:
Is this just a character range? I thought they kept adding emojis all the time? or is `\udfff` the upper limit?